NewEvaluation.css

```diff
@@ -54,8 +57,29 @@ body{
 }
 
 
+.input-candidate{
+	width:33%;
+	border: 0;
+	padding: 0;
+	vertical-align: middle;
+	height: 2em;
+	/*box-shadow: 0px 1px 0px #888888; */
+}
+.input-candidate::placeholder{
+		border-bottom: 1px solid lightgrey;
+		columns: lightgrey;
```

Comment:
Also, particularly interesting usage on a placeholder... Can you describe what you were trying to achieve here?